mvrx/src/main/kotlin/com/airbnb/mvrx/BaseMvRxViewModel.kt

```diff
@@ -167,35 +167,37 @@ abstract class BaseMvRxViewModel<S : MvRxState>(
      * For ViewModels that want to subscribe to itself.
      */
     protected fun subscribe(subscriber: (S) -> Unit) =
-        stateStore.observable.subscribeLifecycle(null, subscriber)
+        stateStore.observable.subscribeLifecycle(null, false, subscriber)
```

Comment:
ViewModels subscriptions don't have to worry about lifecycles, so this is not relevant.